lib/screenplay/jobs/takeover_tool_testing_job.ex

```diff
@@ -0,0 +1,137 @@
+defmodule Screenplay.Jobs.TakeoverToolTestingJob do
+  @moduledoc """
+  Module that executes automated testing for the OFM Takeover Tool.
+  Job tests two things:
+  1. We have the ability to write to and delete from the test folder `ZZZ-MBTA-TEST`
+  2. We see all station folders that we expect to see depending on the screens located at the station.
+  """
+
+  alias Screenplay.Outfront.SFTP
+
+  require Logger
+
+  @test_sftp_directory_name "ZZZ-MBTA-TEST"
+
+  def run do
+    conn = start_connection()
+
+    try do
+      test_creating_and_removing_images(conn)
+      all_directories_exist?(conn)
+    rescue
+      e ->
+        message = "[takeover_tool_testing sftp_connection_error] #{inspect(e)}"
+        Logger.error(message)
+        Sentry.capture_message(message, level: "error")
+    after
+      sftp_client_module().disconnect(conn)
+    end
+  end
+
+  # sobelow_skip ["Traversal.FileModule"]
+  defp test_creating_and_removing_images(conn) do
+    local_image_path = Path.join(:code.priv_dir(:screenplay), "takeover_test.png")
```

Comment:
I think we could compile this image data into the app as a module attribute, and then the app wouldn't compile if the image wasn't present as expected (instead of crashing later when the job runs). Freehanded so idk if this exact code works:

```elixir
@test_image :code.priv_dir(:screenplay) |> Path.join("takeover_test.png") |> File.read!
```

Technically we should also specify this as an [`@external_resource`](https://hexdocs.pm/elixir/Module.html#module-external_resource) of the module, though it's unlikely to change much.

Reply:
Hmm, that's a cool idea. Love it.